server/@types/viewModels/index.d.ts

```diff
@@ -101,6 +101,7 @@ declare module 'viewModels' {
     courseStartDate: Date
     courseCompleted: boolean
     courseCompletionDate?: Date
+    grade?: string
     source: 'CURIOUS'
```

Comment:
Its always Curious at the moment.
As I understand it, its not captured as part of the CIAG induction. I believe they might display it from Curious, and they _might_ save it in their database (I dont know that for sure), but as far as I know its mastered/owned in Curious.

In respect of our integration we are reading it from Curious.

We have future stories (yet to be written or refined etc yet) where we allow adding and updating qualifications. We know that as far as our project is concerned Curious is read-only, so when we add/edit qualifications it must be in some other API/database (probably ours, but yet to be defined)
The point is, when we get to that point, an In Prison qualification could have been read from Curious, or it could have be read from another API. Hence the `source` field